applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -188,8 +188,16 @@ class ExplicitCentralDifferencesScheme
         mTime.Previous = mTime.Current - mTime.Delta;
         mTime.PreviousMiddle = mTime.Current - 1.5 * mTime.Delta;
 
-        if (!BaseType::SchemeIsInitialized())InitializeExplicitScheme(rModelPart);
-        else SchemeCustomInitialization(rModelPart);
+        /// Working in 2D/3D
+        const auto it_elem_begin = rModelPart.ElementsBegin();
+        const auto& r_geom = it_elem_begin->GetGeometry();
+        const SizeType dim(r_geom.WorkingSpaceDimension());
```

Comment:
PROCESSINFO!!!

dude this is like the 5th time or so I give the same comment :D

Reply:
Do you mean the DOMAIN_SIZE?, I am afraid of the not definition of the DOMAIN_SIZE

Reply:
then check (I'd say in `Check`) if it is defined and has an appropriate Value